openapi.yaml

```diff
@@ -1,190 +1,432 @@
+          type: string
+        - in: query
+          name: zipCode
+          description: Filter parks by zip code
           required: false
           type: string
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/ParksResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+    post:
+      summary: Creates a new park
+      tags:
+        - parks
+      operationId: createPark
+      parameters:
+      - $ref: '#/parameters/parkBody'
+      responses:
+        '201':
+          description: Successfully created new park
+          schema:
+            $ref: '#/definitions/ParkResult'
         '400':
-          description: Bad request
+          description: Invalid park object
           schema:
             $ref: '#/definitions/ErrorResult'
         '500':
           description: Internal server error
           schema:
             $ref: '#/definitions/ErrorResult'
-  /pets/{id}:
+  /parks/{parkId}:
     get:
-      summary: Info for a specific pet
+      summary: Lists info for a specific park
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - parks
+      description: Get park by unique ID
+      operationId: getParkById
       parameters:
-        - $ref: '#/parameters/authorization'
-        - name: id
-          in: path
-          description: Unique Pet ID
-          required: true
-          type: string
+        - $ref: '#/parameters/parkIdPath'
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetResult'
+            $ref: '#/definitions/ParkResult'
         '404':
-          description: PetResource not found
+          description: Park resource not found
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+    patch:
+      summary: Update information of a specific park
+      tags: 
+        - parks
+      description: Update park by unique ID
+      operationId: updateParkById
+      parameters:
+        - $ref: '#/parameters/parkIdPath'
+        - $ref: '#/parameters/parkBody'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/ParkResult'
+        '400':
+          description: Incorrect syntax in request, or required fields missing
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '404':
+          description: Park resource not found
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
```

Comment:
It looks like the PATCH request could return a 409: https://jsonapi.org/format/#crud-updating-responses-409